python/paddle/v2/fluid/tests/test_multihead_attention.py

```diff
@@ -58,7 +58,7 @@ def run_program(self):
         """Run the test program.
         """
         places = [core.CPUPlace()]
-        if core.is_compile_gpu():
```

Comment:
Curious how come other test pass for this line?

Reply:
shouldn't have, because my build failed at this test. not sure how yuyang's change passed.

Reply:
https://github.com/PaddlePaddle/Paddle/pull/7817 this is his pull

Reply:
Thanks!

Reply:
just figured that his build didn't include this test.

Reply:
I see, thanks!